CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+project(TOY_MT_FRAMEWORK)
+cmake_minimum_required(VERSION 2.8)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules")
+find_package(OpenMP)
+find_package(TBB)
+find_package(CppUnit)
+find_package(Boost)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
+
+add_subdirectory(BusyWaitCalibration)
+add_subdirectory(DispatchProcessingDemo)
```

Comment:
dispatch is Apple's "Grand Central Dispatch" system:
https://developer.apple.com/library/ios/documentation/Performance/Reference/GCD_libdispatch_Ref/#//apple_ref/doc/uid/TP40008079-CH2-SW2

This comes built-in for OS X. On Linux, I was using the open source version:http://libdispatch.macosforge.org